lisa/sut_orchestrator/azure/README.rst

```diff
@@ -0,0 +1,36 @@
+Azure Deployment Template
+==========================
+
+`Bicep <https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/>`__ provides a more concise and readable way to define Azure infrastructure compared to ARM templates. However, as of now, the Azure Python SDK doesn't support direct deployment of Bicep templates. Therefore, we need to generate an ARM template from the Bicep template and deploy it using the Azure SDK.
+
+To achieve this, follow these steps:
+
+1. **Update Bicep Template**: Update the `arm_template.bicep` file with your desired infrastructure configuration.
+
+2. **Generate ARM Template**: Run the following command to compile the Bicep template into an ARM template:
+   
+   .. code-block:: bash
+   
+      az bicep build -f .\arm_template.bicep
+
+   Make sure you have the Bicep CLI installed. You can install it `here <https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-cli>`__. Put bicepconfig.json in the same directory as the Bicep template.
+
+3. **Deployment Options for Debugging**:
+   
+   - **Deploy with ARM Template and Parameters**: Use the following commands to deploy the generated ARM template with parameters:
+   
+     .. code-block:: bash
+     
+        az group create -l westus3 -n biceptest
```

Comment:
Is it really needed? And it's hard to follow, because it needs a parameters file, but not exists here. You can specify after that, run LISA as normal see if any error and new changes applied or not.

Reply:
Actually, comparing with previous steps, one more step is to build json before call LISA.

Reply:
I add it under Deployment Options for Debugging section

Reply:
It's not recommended to debugging with az commands, because the reader may not familiar with az commands, and it doesn't save time or provide more information by using lisa directly.

Reply:
updated